packages/statemanager/test/stateManager.spec.ts

```diff
@@ -281,4 +284,36 @@ describe('StateManager -> General', () => {
     zeroAccount = await newPartialStateManager2.getAccount(Address.zero())
     assert.ok(zeroAccount!.nonce === zeroAddressNonce)
   })
+  it.skipIf(isBrowser() === true)(
+    'should create a statemanager fromProof with opts preserved',
+    async () => {
+      const trie = await Trie.create({ useKeyHashing: false })
```

Comment:
Ok, so this `trie` uses `useKeyHashing: false` in order to check that we can retrieve the original data, right? (Just checking). If we would use `useKeyHashing: true` and then use `useKeyHashing: false` in the trie and state manager below here then this would lead to incorrect results.

Reply:
Correct.